src/arch/ia32/descriptor/segment.rs

```diff
@@ -279,6 +298,10 @@ impl fmt::Display for SegmentDescriptor {
     }
 }
 
+impl Default for SegmentDescriptor {
```

Comment:
I'm wondering if introducing a `ConstDefault` trait would be a good idea, it would inherit default.

Reply:
It seems that *functions in traits cannot be const* so I do not know if that would be possible.
Actually I am considering to just rename these `const_default` into `default` and just no inherits Default trait